src/server/index.jsx

```diff
@@ -148,6 +148,7 @@ server
         data.toggles = toggles;
         data.path = urlPath;
         data.timeOnServer = Date.now();
+        data.canAdvertise = headers['bbc-adverts'];
```

Comment:
Looks good Khoa, just one comment. Will this header come through from mozart as a string? Do we need to make sure we cast it to a boolean before passing it into the application?

Reply:
I just ran mozart locally with simorgh. and the BBC-Adverts header gets converted to a string when passed as header 🤕 . I've added a check for the string 'true' here: https://github.com/bbc/simorgh/pull/7205/commits/3b03191cb831cf3eb44b20153c0cd6c60a3b1b5a